app.ts

```diff
@@ -60,12 +69,12 @@ function startGateway () {
   gw = new Gateway(settings.gateway, zwave, mqtt)
 }
 
-app.startSocket = function (server) {
+app.startSocket = function (server: any) {
```

Comment:
Wow, that's bad. I think you'll get better results by converting the require statements to imports and let VSCode deduce the types.

Reply:
@AlCalzone You are right that this is bad if it would already be the finished migration ;-).

I guess this is on purpose for the chosen migration strategy "All-in migration" (see <https://medium.com/airbnb-engineering/ts-migrate-a-tool-for-migrating-to-typescript-at-scale-cd23bfeb5cc>) that states the following advantages:
* Consistency across a project
* Fixing just one type is much easier than fixing the whole file

This way you can migrate the whole code base to typescript without having to support a hybrid JavaScript/TypeScript build system and code base.

The next step would of course be introducing the types, but that is something that might take a bit longer ...